VisualStudio.gitignore

```diff
@@ -215,3 +215,15 @@ FakesAssemblies/
 **/*.Server/GeneratedArtifacts
 **/*.Server/ModelManifest.xml
 _Pvt_Extensions
+
+# Execution reports created Ranorex
+*.nlog*
+*.rxlog*
+*.rxlog
+*_trace*
+*/*.rxuser
+*.rxuser
+*.rximg
+Report
```

Comment:
In some other scenarios, rules can be committed but then commented out, so users can opt-in to it, e.g.

```
# Uncomment if necessary however generally it will be regenerated when needed
 #!**/packages/repositories.config
```

This helps to explain when specific rules might be necessary.